error/src/lib.rs

```diff
@@ -1402,6 +1392,26 @@ impl CaliptraError {
             0x000E005A,
```

Comment:
Not exactly related to this PR, but it looks like we got a bit out of sync between the 1.x and 2.x error codes: https://github.com/chipsalliance/caliptra-sw/blob/main/error/src/lib.rs#L468. We can do it in a follow-up PR, but we should align those.